examples/createClient.js

```diff
@@ -30,7 +30,7 @@ const writeFile = async client => {
 }
 
 const readFile = async client => {
-  const recordID = '6987f7c7-6016-495a-9156-34d66ababc55'
+  const recordID = '5d98c2f4-22e8-484a-be8d-94ad50940ae8'
```

Comment:
Could this be parsed from the writeFile response and automatically called? Then the example could double as a sort of integration test.